specs/language/introduction.tex

```diff
@@ -319,3 +344,9 @@
 \gls{lane}s executing on the device. Constant memory is read-only, and an
 implementation can assume that constant memory is immutable and cannot change
 during execution.
+
+\SubSub{Constant Memory}{Intro.Memory.Spaces.Overlap}
+
+\p The \textbf{Thread} and \textbf{Thread Group} memory spaces may not overlap
```

Comment:
Is paragraph just stating part of what was said in the paragraph starting on line 311? 
Nit: I was initially confused by the use of both 'memory location' and 'address'; I had just glazed over the fact that they are defined to be the same thing. It might be nice to use one or the other.